sonar-project.properties

```diff
@@ -0,0 +1,16 @@
+sonar.projectKey=LedgerHQ_app-icp
+sonar.organization=ledger
+
+# This is the name and version displayed in the SonarCloud UI.
+sonar.projectName=app-icp
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+sonar.sources=.
```

Comment:
This feels a bit vague. Shouldn't the scan be restricted to `app/src` (and some other) rather than the whole repository ? Or you believe tests, fuzzing, ... code should be included in the scan?

Reply:
I think test files and fuzzers should reach the same quality than the rest of the project, and then should be included.